services/web/server/src/simcore_service_webserver/templates/common/close_account.jinja2

```diff
@@ -0,0 +1,31 @@
+🐼 Closing your account in {{ host }} and best wishes
+<!--
+TEST host:{{ host }}
+TEST name:{{ name }}
+TEST support_email:{{ support_email }}
+TEST retention_days:{{ retention_days }}
+-->
```

Comment:
make sure to strip these when rendering the email

Reply:
This already happens before rendering. All comments added in templates and automatically removed
https://github.com/ITISFoundation/osparc-simcore/blob/f6c6d69cdbcf05f048b4bbd2668c67970dbb6b78/services/web/server/src/simcore_service_webserver/email/_core.py#L207